src/main/java/io/debezium/connector/vitess/VitessType.java

```diff
@@ -28,27 +40,59 @@ public int getJdbcId() {
         return jdbcId;
     }
 
+    public List<String> getEnumValues() {
+        return enumValues;
+    }
+
+    public boolean isEnum() {
+        return enumValues != null && enumValues.size() != 0;
```

Comment:
As per above, you then can simply do return !enumValues().isEmpty().

Reply:
Good point. Addressed